### PR TITLE
Add function that can get models by prefix

### PR DESCRIPTION
**Pull request by Ujcgjlm**
A useful feature has been added:
A function can retrieve models with a matching prefix in a particular module

Reduced cyclomatic complexity of the existing function: get_all_auto_configured_models

### CODE REVIEW

**Comment by ArthurZucker** [comment]
Hey! Feel free to ping me or @ydshieh for a review when this is ready!

**Comment by Ujcgjlm** [comment]
> Hey! Feel free to ping me or @ydshieh for a review when this is ready!

I apologize for not responding. It's done

File: utils/check_repo.py [hunk 1]
```diff
@@ -600,21 +600,24 @@ def check_all_models_are_tested():
         raise Exception(f"There were {len(failures)} failures:\n" + "\n".join(failures))
 
 
+def get_models_with_prefix(model: types.ModuleType, attr_prefix: str) -> Set[str]:
```
Comment (ydshieh) [inline utils/check_repo.py:603]: The first argument named `model`, but it is actually a module of auto model. Maybe just name it as `module`.

File: utils/check_repo.py [hunk 2]
```diff
@@ -600,21 +600,24 @@ def check_all_models_are_tested():
...
+    """Return the set of all models in class that start with prefix"""
```
Comment (ydshieh) [inline utils/check_repo.py:604]:
```suggestion
    """Return the set of all models defined in the auto mappings in a module of auto models."""
```

File: utils/check_repo.py (hunk 1 above)
Comment (ydshieh) [reply utils/check_repo.py:603]: name function as `get_models_in_auto_mappings` might be better.

The prefix is not the prefix of models, but of the auto mappings

**Pull request closed without merging**